.github/workflows/build_and_publish.yaml

```diff
@@ -28,8 +28,9 @@ jobs:
     outputs:
       tag: ${{ steps.tag.outputs.new_version }}
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v2
```

Comment:
Since the default branch on the actions/checkout repo is now `main`, `@master` was stuck back in July 2020.  
According to this comment, it was because the current version at that time checked the code out without any history
https://github.com/opensafely-core/cohort-extractor/issues/232#issuecomment-654774782

Since we can do `fetch-depth: 0` to get all history now, I think v2 should work for us.